build.sbt

```diff
@@ -987,8 +997,12 @@ lazy val kernelGroup = project
     // crossScalaVersions must be set to Nil on the aggregating project
     crossScalaVersions := Nil,
     publishArtifact := false,
-    publish / skip := false
-  )
+    publish / skip := false,
+    unidocSourceFilePatterns := {
```

Comment:
this will generate a unified kernel api and default API docs that we will upload to our website.